code/modules/urist/items/vgitems.dm

```diff
@@ -80,3 +80,73 @@
 	w_class = ITEM_SIZE_SMALL
 	obj_flags = OBJ_FLAG_CONDUCTIBLE
 	insultmsg = list("HONK!", "HELP, SHITCURITY!", "HELP MAINT", "SHITSEC", "I AM IN A CONSTANT STATE OF SUFFERING, EACH EON SPENT ON THIS PLANE OF EXISTENCE ECHOES ONLY PURE PAIN TO MY MORTAL COIL. GOD HAS DIED, AND HE HAS ONLY LEFT ME SUFFERING. ", "MY FUNNY BONE!")
+
+//lottery scratchers - west3436 of /vg/station13
+/obj/item/toy/lotto_ticket
+	name = "scratch-off lotto ticket"
+	desc = "A scratch-off lotto ticket."
+	icon = 'icons/urist/items/tgitems.dmi' //i see no reason to change this
+	w_class = ITEM_SIZE_TINY
+	var/revealed = FALSE
+	var/iswinner = FALSE
+	var/prize_multiplier
+	var/winnings = 0
+	var/list/prizelist = list(100000,50000,10000,5000,1000,500,250,100,50,20,10,5,4,3,2,1)
+	var/list/problist = list(0.0001, 0.0002, 0.001, 0.002, 0.01, 0.02, 0.04, 0.2, 1, 2.5, 5, 10, 12.5, 17, 20, 25)
+
+/obj/item/toy/lotto_ticket/Initialize()
+	..()
+	pixel_y = rand(-8, 8) * PIXEL_MULTIPLIER
+	pixel_x = rand(-9, 9) * PIXEL_MULTIPLIER
+
+/obj/item/toy/lotto_ticket/proc/scratch(var/input_prize_multiplier)
+	var/tuning_value = 1/5 //Used to adjust expected values.
+	var/profit = 0
+	for(var/prize = 1 to problist.len)
+		if(prob(problist[prize]))
+			profit = prizelist[prize]*input_prize_multiplier*tuning_value
+			return profit
+
+/obj/item/toy/lotto_ticket/attackby(obj/item/S, mob/user)
+	if(!revealed)
+		if(is_sharp(S) || istype(S, /obj/item/material/coin))
+			if(do_after(user, src, 1 SECONDS))
+				src.revealed = TRUE
+				src.update_icon()
+				to_chat(user, "<span class='notice'>You scratch off the film covering the prizes.</span>")
+				winnings = scratch(prize_multiplier)
+				if(winnings)
+					src.iswinner = TRUE
+		else
+			to_chat(user, "<span class='notice'>You need to use something sharp to scratch the ticket.</span>")
+	else
+		to_chat(user, "<span class='notice'>The film covering the prizes has already been scratched off.</span>")
+
+/obj/item/toy/lotto_ticket/examine(mob/user)
+	if(revealed)
+		if(iswinner)
+			to_chat(user, "<span class='notice'>This one is a winner! You've won [winnings] thalers.</span>")
```

Comment:
This should be condensed into a single statement for better readability. No real reason to split it like this. You also need a call to parent to display the original item description.
```suggestion
	. = ..()
	if(revealed && iswinner)
		to_chat(user, SPAN_NOTICE("This one is a winner! You've won [winnings] thalers."))
```
If the previous suggestion to remove `iswinner` is accepted, you'll just swap `iswinner` out for `winnings`